src/SprykerEco/Yves/Payone/Theme/default/components/molecules/payone-credit-card/payone-credit-card.ts

```diff
@@ -139,7 +159,10 @@ export default class PayoneCreditCard extends Component {
 
     get isCurrentPaymentMethod(): boolean {
```

Comment:
Have you checked it with linters? Please add change return type to `bollean | null`.